### DISCUSSION TITLE
Set value L_EDDY

### DISCUSSION THREAD

**Discussion by mpachera** [firemodels/fds#12089] [Q&A]
Dear,

I am implementing a turbulent flow as boundary condition where I know the VEL_RMS, but I don't know how I should set the L_EDDY. I looked around the validation and verification repositories and I saw that you use between 1/4 or 1/10 of the characteristic length of the inlet. I would like to understand if there's any guidance about the setup of the parameter.

Thanks in advance,

Kind regards,

Matteo

**Comment by rmcdermo** [comment]
I don't have any guidance beyond what we have written in the user guide.
L_EDDY is a parameter in Jarrin's eddy size distribution.  It obviously
needs to be bigger than a few grid cells otherwise there is nothing to
sample.  The upper range will depend on the flow problem, and (like
PATH_LENGTH) you should test sensitivity to this (and N_EDDY) for your
case.  But a reasonable starting point is the integral length scale of the
inlet turbulence---what are the "large eddies" upstream of the boundary you
are trying to capture?  Remember, in reality turbulence is a coherent
solution to the Navier-Stokes equations.  SEM is a major shortcut, and you
should not expect perfection.  The verification cases where Jarrin
reconstructs Reynolds stresses are highly idealized flows.

**Reply by mpachera** [reply]
I'm working on a wind model. I have a large scale wind model and I use its results for a more local model as boundary conditions. I want to implement the local velocity by also the turbulence. The domain is 80 m high and I was thinking to use something like 10 m because it's also representative of the height of the buildings in the surroundings.

**Comment by rmcdermo** [comment]
10 m sounds like a reasonable starting point.  But boiling all of
turbulence down to one turbulence intensity and length scale is just an
approximation.  What is your metric for correctness?  You likely need to
run a few cases covering the ranges of feasibility, say 2 m on the low end
to 20 m on the high end to see what difference it makes in your answer.

**Reply by mpachera** [reply]
but can I kind of measure the eddy's length in my large scale simulation? I could use that to ensure that I have a similar flowfield in the small scale model.

**Comment by rmcdermo** [comment]
Well, there is no direct output for this.  It's like measuring flame
height.  You should just put a slice of your velocity components across the
inlet of the face and look to see if the length scales are what you want.

**Reply by mpachera** [reply]
I looked again at the UG and from what I get my L_EDDY is the maximum size of the eddy, but also the distance to the wall where I let my eddies to develop. So if I understand correctly I can use a larger value that includes all my eddies' lengths.

**Reply by rmcdermo** [reply]
L_EDDY is just a parameter in the eddy sampling distribution.  Yes, multiple eddy sizes are sampled.
